src/SSVOpenHexagon/Core/HexagonGame.cpp

```diff
@@ -473,6 +476,26 @@ void HexagonGame::death(bool mForce)
     assets.playSound("gameOver.ogg", ssvs::SoundPlayer::Mode::Abort);
     runLuaFunctionIfExists<void>("onDeath");
 
+    std::string nameStr = levelData->name;
```

Comment:
Bring this to down to line 492, and leave an empty line between the PB stuff and the rest. Easier to group visually